libs/go/libp2p_node/acn/utils.go

```diff
@@ -0,0 +1,129 @@
+/* -*- coding: utf-8 -*-
+* ------------------------------------------------------------------------------
+*
+*   Copyright 2018-2019 Fetch.AI Limited
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       http://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*
+* ------------------------------------------------------------------------------
+ */
+package acn
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"github.com/rs/zerolog"
+	proto "google.golang.org/protobuf/proto"
+)
+
+var logger zerolog.Logger = zerolog.New(zerolog.ConsoleWriter{
+	Out:        os.Stdout,
+	NoColor:    false,
+	TimeFormat: "15:04:05.000",
+}).
+	With().Timestamp().
+	Str("package", "AeaApiACN").
+	Logger()
+
+const CurrentVersion = "0.1.0"
+
+type Pipe interface {
+	Connect() error
+	Read() ([]byte, error)
+	Write(data []byte) error
+	Close() error
+}
+
+func DecodeACNMessage(buf []byte) (error, *AeaEnvelope, *Status) {
```

Comment:
We should adhere to standard go approach, I think error is last return type by convention (e.g. https://gobyexample.com/errors).

Reply:
done